File: java/darts/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        double radius = Math.sqrt(xOfDart * xOfDart + yOfDart * yOfDart);
        int score;

        if     (radius > 10.0) score = 0;
        else if (radius > 5.0) score = 1;
        else if (radius > 1.0) score = 5;
        else score = 10;

        return score;
    }
}

File: fsharp/darts/Darts.fs
module Darts

let score (x: double) (y: double): int =
    let radius = sqrt (x*x + y*y)

    if   radius <= 1.0 then 10
    elif radius <= 5.0 then 5
    elif radius <= 10.0 then 1
    else 0
